### ISSUE TITLE
Use Frigate alerts/detections to help find clips faster

### ISSUE THREAD

**Issue by PablaV** [dermotduffy/advanced-camera-card#1502]
With the updates to 14.0 it would be nice to be able to view Alerts and Detections like you can in the WebUI. Currently the Frigate Card shows events separately in a similar fashion to how things were in <13.0. IMO the combining of multiple events into one alert has really cut down on the time it takes to find the clip you're interested in.

**Comment by conorlap** [comment]
+1 for this, would be really useful!

**Comment by dnestico** [comment]
+1 pls

**Comment by tremby** [comment]
As a stopgap until this is available, is there a way to at least set a default filter on the clips page? I can filter it to only clips where particular zones were entered, but this is a pain to do every time.

**Comment by dermotduffy** [comment]
There is a fresh preview release available with Frigate 'detection' functionality, I'd check that out: https://github.com/dermotduffy/advanced-camera-card/releases/tag/v8.0.0-rc.1

> As a stopgap until this is available, is there a way to at least set a default filter on the clips page? I can filter it to only clips where particular zones were entered, but this is a pain to do every time.

Yes. See [documentation](https://card.camera/#/configuration/cameras/engine?id=frigate).

**Comment by tremby** [comment]
> > As a stopgap until this is available, is there a way to at least set a default filter on the clips page? I can filter it to only clips where particular zones were entered, but this is a pain to do every time.
> 
> Yes. See [documentation](https://card.camera/#/configuration/cameras/engine?id=frigate).

 It seems you mean setting `frigate.labels` and `frigate.zones`. The docs aren't clear that these will be default filters. It sounds like it's to define what the *available* labels/zones for filtering are.

That helps; thank you.

**Comment by dermotduffy** [comment]
> It seems you mean setting frigate.labels and frigate.zones. The docs aren't clear that these will be default filters. It sounds like it's to define what the available labels/zones for filtering are.

Yes, these are the default filters. You can obviously change them when you're browsing the media gallery (in the filter drawer on the right hand side of the media gallery), but whatever you configure in the config is the default for the gallery and for thumbnail fetching more generally for that camera.

Will tweak the docs to improve this clarity.
